mlc_llm/core.py

```diff
@@ -402,6 +403,10 @@ class BuildArgs:
             "action": "store_true",
         },
     )
+    paged_kv_cache_type: str = field(
+        default="vllm",
+        metadata={"help": "The type of paged KV cache, either vllm or flash-decoding"},
```

Comment:
This new option makes `--use_vllm_attention` obsolete. Since removing it is a breaking change, I'll do that later when I integrate Flash-Decoding into `mlc-serve`. @sunggg